### ISSUE TITLE
0.3.136 SandboxViolation

### ISSUE THREAD

**Issue by AxelNennker** [hyperledger/indy-client#229]
The error message seems to be very off: This is Ubuntu - so no .exe

Axel

ignisvulpis@namenlos:~/development/sovrin/sovrin-client$ python setup.py pytest
running pytest
Searching for sovrin-node-dev==0.3.136
Reading https://pypi.python.org/simple/sovrin-node-dev/
Downloading https://pypi.python.org/packages/8d/2f/c752ef03349a3868b759a50df33d110551b99ef42c6c9420813e6d78d8ff/sovrin-node-dev-0.3.136.tar.gz#md5=9a7ceaa59ba1a2a26d0ad1f244b43b2b
Best match: sovrin-node-dev 0.3.136
Processing sovrin-node-dev-0.3.136.tar.gz
Writing /tmp/easy_install-o10rsyac/sovrin-node-dev-0.3.136/setup.cfg
Running sovrin-node-dev-0.3.136/setup.py -q bdist_egg --dist-dir /tmp/easy_install-o10rsyac/sovrin-node-dev-0.3.136/egg-dist-tmp-ai2gp8id
error: Setup script exited with error: SandboxViolation: open('/home/ignisvulpis/.sovrin/nssm_original.exe', 'wb') {}

The package setup script has attempted to modify files on your system
that are not within the EasyInstall build area, and has been aborted.

This package cannot be safely installed by EasyInstall, and may not
support alternate installation locations even if you run its setup
script by hand.  Please inform the package's author and the EasyInstall
maintainers to find out if a fix or workaround is available.
ignisvulpis@namenlos:~/development/sovrin/sovrin-client$

**Comment by AxelNennker** [comment]
ignisvulpis@namenlos:~/development/sovrin/sovrin-client$ easy_install
easy_install      easy_install-2.7  easy_install3     easy_install-3.4  easy_install-3.6  
ignisvulpis@namenlos:~/development/sovrin/sovrin-client$ easy_install

ignisvulpis@namenlos:~/development/sovrin/sovrin-client$ python --version
Python 3.6.1
ignisvulpis@namenlos:~/development/sovrin/sovrin-client$

**Comment by dhh1128** [comment]
@AxelNennker the reference to an .exe is caused by a pip package for a sovrin dependency. I am thinking about how I can find a developer who can help you on this. Are you doing development against Sovrin, or are you just trying to install it so you can use it?

**Comment by AxelNennker** [comment]
First I want to install it so I can use it. Later we might do some development too. 
The is the other issue#226 that stopped me trying the canned version so I tried the latest and greatest fresh from the repository.

//Axel

hey @peacekeeper , @windley

**Comment by dhh1128** [comment]
Axel, you are following a workflow that is unfamiliar to me. When I want to install sovrin, I follow instructions like the ones in this doc: https://docs.google.com/document/d/1Da6adGCoRhCntQwPQPzz1H1dJRHAmgyyDi0k01ncvuw/edit.

I have never run pytest and seen it install sovrin-node-dev as part of its test fixture. Nor have I (or anyone on the dev team) worked with easy_install; we use pip by preference. My usual workflow is to get all the source code into sibling directories, and then run pytest on it. Most developers on the team work in pycharm and configure the IDE with pytest as the active testrunner. (We do have a few people who use alternate workflows and editors; I am just less familiar with their procedures...)

Can you provide some details about what install procedures you followed?

**Comment by AxelNennker** [comment]
First I followed this "getting started": https://sovrin.org/wp-content/uploads/2017/04/getting-started.pdf
This ended in https://github.com/sovrin-foundation/sovrin-client/issues/226
Then I did a "git pull" on sovrin-client and reran the setup and pytest as described in setup.md
https://github.com/sovrin-foundation/sovrin-client/blob/stable/setup.md
Which ended in this issue.

I never ran easyinstall by hand and I guess setup.cfg/py is borked for Ubuntu.

My plan was/is to try the sovrin-client against "some" sovrin network. I followed the getting_started and tried the test network...

Setting up my own network is a task for later.
